libkbfs/interfaces.go

```diff
@@ -132,6 +132,18 @@ type Node interface {
 	// `inner.Readonly()` if it decides not to return `true` on its
 	// own.
 	Readonly(ctx context.Context) bool
+	// ShouldCreateMissedLookup is called for Nodes representing
+	// directories, whenever `name` is looked up but is not found in
+	// the directory.  If the Node decides a new entry should be
+	// created matching this lookup, it should return `true` as well
+	// as a context to use for the creation, the type of the new entry
+	// and the symbolic link contents is the entry is a Sym; the
+	// caller should then create this entry.  Otherwise it should
+	// return false.  An implementation that wraps another `Node`
+	// (`inner`) must return `inner.ShouldCreateMissedLookup()` if it
+	// decides not to return `true` on its own.
+	ShouldCreateMissedLookup(ctx context.Context, name string) (
+		bool, context.Context, EntryType, string)
```

Comment:
I assume the last returned argument `string` is the content of the symlink target. Perhaps use named return type here to make the meaning clear from method signature?